client/src/components/HomePage/modal/ModalContext/index.jsx

```diff
@@ -37,20 +39,30 @@ class Provider extends Component {
       this.setState({ popUpMessage });
     },
 
-
     storeValue: (event) => {
       const { target } = event;
-      const { value } = target;
+      const {
+        value,
+        name,
+        checked,
+        parentNode,
+      } = target;
       let valutToSave = '';
+      const { data } = this.state;
+      const { skills } = data;
       if (target.type === 'checkbox') {
-        valutToSave = target.checked;
+        if (checked) {
+          skills.push({ [name]: parentNode.innerText });
```

Comment:
textContent is better than innerText. Unless there's hidden text we don't want to show, we should use textContent.